mk/e2e.new.mk

```diff
@@ -32,7 +32,7 @@ UNIVERSAL_E2E_PKG_LIST ?= ./test/e2e_env/universal
 MULTIZONE_E2E_PKG_LIST ?= ./test/e2e_env/multizone
 GINKGO_E2E_TEST_FLAGS ?=
 GINKGO_E2E_LABEL_FILTERS ?=
-GINKGO_TEST_E2E=$(GINKGO_TEST) -v --slow-spec-threshold 30s $(GINKGO_E2E_TEST_FLAGS) --label-filter="$(GINKGO_E2E_LABEL_FILTERS)"
```

Comment:
why do we remove it?

Reply:
`--slow-spec-threshold` is deprecated because ginkgo doesn't warn on slow tests anymore (they came to the same conclusion as us, it's just more noisy than useful)